app/routes/missions/icecube.mdx

```diff
@@ -5,63 +5,158 @@ handle:
 
 import logo from '~/img/icecube-logo.png'
 
-# IceCube
+# IceCube Neutrino Observatory
 
 <img
   src={logo}
   alt="IceCube logo"
-  width="640"
-  height="181"
+  align="right"
+  width="150"
+  height="135"
```

Comment:
These numbers don't match the dimensions or the aspect ratio of the image file.

Reply:
fixed